src/document/json/object.ts

```diff
@@ -115,16 +119,20 @@ export class JSONObject extends JSONContainer {
     return clone;
   }
 
-  public *getDescendants(): IterableIterator<JSONElement> {
+  public getDescendants(
+    callback: (elem: JSONElement, parent: JSONContainer) => boolean,
+  ): IterableIterator<JSONElement> {
```

Comment:
The implementation doesn't seem to return anything. Should we change it to `void`?

```suggestion
  public getDescendants(
    callback: (elem: JSONElement, parent: JSONContainer) => boolean,
  ): void {
```